src/js/components/tablero/NumberCard.jsx

```diff
@@ -103,7 +104,7 @@ const NumberCard = SortableElement(({
               translate(cardTitle.code, cardTitle.message)
              : cardTitle}
           </span>
-          <span className="result-card"> {cardNumber.toLocaleString()} </span>
+          <span className="result-card"> {cardNumberType === 'number' ? cardNumber.toLocaleString() : `${cardNumber.toLocaleString()}USD`} </span>
```

Comment:
Should not this support other currencies?

Reply:
The ticket specified USD. Should we implement it for other currencies?

Reply:
@bamoure Yeah we should pull the default currency from the configuration. Let me know if you need help with that.

Reply:
@awalkowiak Can you help add the default currency to the config endpoint?

Reply:
Sure I'll look into that today

Reply:
@jmiranda done